events/consts.py

```diff
@@ -6,3 +6,9 @@ class NotificationType:
     OCCURRENCE_CANCELLED = "occurrence_cancelled"
     OCCURRENCE_REMINDER = "occurrence_reminder"
     OCCURRENCE_FEEDBACK = "occurrence_feedback"
+
+
+notification_types_that_needs_communication_acceptance = (
```

Comment:
There's a mismatch between noun's and verb's grammatical number i.e.
`types` is plural, `needs` is singular, they should go together so if `types` is plural then verb should be in this case in plural form i.e. `need`:
```suggestion
notification_types_that_need_communication_acceptance = (
```